e2e/cypress/integration/channel/archived_channels_spec.js

```diff
@@ -156,4 +158,319 @@ describe('Leave an archived channel', () => {
             cy.get('#search-items-container .no-results__wrapper').should('be.visible');
         });
     });
+
+    it('MM-T1696 - When clicking "More..." for Public and Private channels, no options for archived channels are shown when the feature is disabled', () => {
+        cy.apiAdminLogin();
+        cy.apiUpdateConfig({
+            TeamSettings: {
+                ExperimentalViewArchivedChannels: false,
+            },
+        });
+
+        // # Create public channel
+        cy.apiCreateChannel(testTeam.id, 'channel', 'channel').then(({channel}) => {
+            // # Visit the channel
+            cy.visit(`/${testTeam.name}/channels/${channel.name}`);
+
+            // # Archive the channel
+            cy.uiArchiveChannel();
+
+            // # Open more modal
+            cy.get('#publicChannelList').should('be.visible').within(() => {
+                cy.findByText('More...').scrollIntoView().should('be.visible').click();
+                cy.get('#channelsMoreDropdown').should('not.be.visible');
+
+                // # Modal should not contain the created channel
+                cy.get('#moreChannelsList').should('not.contain', channel.name);
+            });
+        });
+    });
+
+    it('MM-T1697 - More... for Public channels shows archived channels option', () => {
+        cy.apiAdminLogin();
+        cy.apiUpdateConfig({
+            ServiceSettings: {
+                ExperimentalChannelSidebarOrganization: 'default_off',
```

Comment:
Can you add `EnableLegacySidebar: 'true'` here as well or will that break things?
We're moving to that new setting so I want to make sure my changes won't break this test.

Reply:
Oh yeah sure but maybe we should wait for those changes to go in first? In order to keep the tests passing?

Reply:
Sure, we'll just need to remember to fix these later. My PRs won't be going in for a while since we don't want to force the new sidebar on until the auto-closing and new unread category are complete. I'll try to remember when I merge my stuff to do that :P